cypress/integration/functional/wallet/authorization/transactionAuthorization.spec.ts

```diff
@@ -143,13 +143,13 @@ context('Wallet - Transaction Authorization', () => {
 
     it('should be able to exit failed retries', function () {
       Array.from(Array(4), (v, i) => i + 1).forEach(() => {
-        cy.getByTestID('pin_authorize').type('696969').wait(1000)
+        cy.getByTestID('pin_authorize').type('696969').wait(2000)
       })
       cy.url().should('include', 'wallet/onboarding')
     })
 
     it('should clear attempt on success', function () {
-      cy.createEmptyWallet(true)
+      cy.createEmptyWallet(true).wait(4000)
```

Comment:
Spare time for encryption to run, probably too long, depend on hardware.

0 wait time randomly fail on my own machine.